plugins/modules/network_settings_workflow_manager.py

```diff
@@ -1543,16 +1542,18 @@ def get_want_reserve_pool(self, reserve_pool):
             if pool_values.get("ipv4Prefix") is True:
                 if pool_values.get("ipv4Subnet") is None and \
                         pool_values.get("ipv4TotalHost") is None:
-                    self.msg = "missing parameter 'ipv4_subnet' or 'ipv4TotalHost' \
-                        while adding the ipv4 in reserve_pool_details '{0}' element".format(reserve_pool_index + 1)
+                    self.msg = "missing parameter 'ipv4_subnet' or 'ipv4TotalHost' " + \
```

Comment:
ipv4TotalHost or ipv4_total_host

Reply:
self.msg = "Failed to add IPv4 in reserve_pool_details '{0}'. " \
           "Required parameters 'ipv4_subnet' or 'ipv4_total_host' are missing." \
           .format(reserve_pool_index + 1)